packages/host/src/lib/serviceDiscovery/sd-adapter.ts

```diff
@@ -53,13 +53,6 @@ export class ServiceDiscovery {
         return this.topicsController.get(id);
     }
 
-    createTopic(id: TopicId, contentType: ContentType) {
```

Comment:
I would move this arguments structure `(id: TopicId, contentType: ContentType)`, to `createTopicIfNotExist` for readibility and type safety, looks better than "magic" structure as arguments.

Reply:
Can do.